Dockerfile.postgres

```
@@ -0,0 +1,49 @@
+    gcc \
+    python3-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN python3 -m venv /opt/mbslave-venv
+ENV PATH="/opt/mbslave-venv/bin:$PATH"
+
+WORKDIR /usr/src/app
+COPY . .
+
+# Install mbslave and its dependencies
+RUN pip install .
```

Hadolint (2.14.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)